src/java/Employee.java

```diff
@@ -0,0 +1,57 @@
+class Employee {
+    private String FullName;
```

Comment:
The name of the variables should start with a lowercase letter. Please remember that in the future.
Please rename all variables, so the variable `FullName` should be renamed to `fullName`. The rest should be renamed accordingly.